base_layer/core/src/proof_of_work/proof_of_work.rs

```diff
@@ -108,13 +108,18 @@ impl ProofOfWork {
     ///
     /// The total accumulated difficulty is most often used to decide on which of two forks is the longest chain.
     pub fn total_accumulated_difficulty(&self) -> Difficulty {
```

Comment:
Is this function still used?

Reply:
This still gets stored in the chain metadata. Changing this would cause a reorgs in old blocks